adhocracy-plus/config/settings/base.py

```diff
@@ -283,17 +285,18 @@
     "signup": "apps.users.forms.DefaultSignupForm",
     "login": "apps.users.forms.DefaultLoginForm",
 }
-ACCOUNT_LOGIN_ATTEMPTS_LIMIT = 10
-ACCOUNT_LOGIN_ATTEMPTS_TIMEOUT = 300  # seconds
+ACCOUNT_RATE_LIMITS = {"login_failed": "10/5m/ip"}
 ACCOUNT_LOGIN_ON_EMAIL_CONFIRMATION = True
 ACCOUNT_LOGIN_ON_PASSWORD_RESET = True
 ACCOUNT_USERNAME_MIN_LENGTH = 5
 ACCOUNT_USERNAME_REQUIRED = True
-ACCOUNT_PREVENT_ENUMERATION = True
+ACCOUNT_PREVENT_ENUMERATION = "strict"
 SOCIALACCOUNT_AUTO_SIGNUP = False
 SOCIALACCOUNT_EMAIL_VERIFICATION = "none"
 SOCIALACCOUNT_FORMS = {"signup": "apps.users.forms.SocialTermsSignupForm"}
 SOCIALACCOUNT_QUERY_EMAIL = True
+SOCIALACCOUNT_EMAIL_AUTHENTICATION = True
```

Comment:
@m4ra from my testing and research this apparently will only work with providers which verify the user email (which makes sense security-wise). From what I could find out in our case that only is apple and google. For github and facebook it won't work, so I wonder if it makes sense to enable this option at all. Not sure how to proceed here, what do you think?

Reply:
I changed the socialaccount verification to mandatory, so if not verified from provider, the email verification should be sent (this is what I understand from the allauth documentation). Can you pull and have another try? The dummy provider doesn't work in this scenario. It gives error that the field is required, aka the email field, which is a hidden form and from the browser inspect it's value is a long string, I guess it's encrypted?
![image](https://github.com/liqd/adhocracy-plus/assets/5695572/3ee6165c-3904-499a-beaf-a6f0a5caf6e0)